browser/ui/views/infobars/brave_wayback_machine_infobar_contents_view.cc

```diff
@@ -110,14 +115,19 @@ void BraveWaybackMachineInfoBarContentsView::HideInfobar() {
   }
 }
 
-void BraveWaybackMachineInfoBarContentsView::ButtonPressed() {
+void BraveWaybackMachineInfoBarContentsView::FetchURLButtonPressed() {
   if (wayback_url_fetch_requested_)
     return;
   wayback_url_fetch_requested_ = true;
-
+  dont_ask_button->SetVisible(false);
```

Comment:
right now, the "Don't ask me again" is being hidden when the user clicks on the "Check for saved version" button. This behaviour can be seen in video attached for Scenario 2. One more possibility can be disabling that button instead of hiding it.

Reply:
What happens w/o this call?

Reply:
Ah, I see. this `ask button` will be hidden while throbber is running :)

Reply:
Hmm, or how enabling that button? With this, user can hide while waiting saved version?

Reply:
NVM, I think hiding that button would be better :)